### PR TITLE
Prepare repository for Tenzir v4.0.0-rc1

### PR DESCRIPTION
**Pull request by tobim**
The first of probably many release candidates for Tenzir v4.0.

### CODE REVIEW

File: scripts/prepare-release [hunk 1]
```diff
@@ -78,8 +78,8 @@ if [[ true ]]; then
   new_version_name="Tenzir ${new_version_major_minor}"
   jq "map(select(. | startswith(\"${new_version_name%-*}\") | not))" < versions.json > versions.json.tmp
   mv -f versions.json.tmp versions.json
-  rm -rf "versioned_docs/version-${new_version_name%-*}"*
-  rm -rf "versioned_sidebars/version-${new_version_name%-*}"*-sidebars.json
+  rm -rf "versioned_docs/version-${new_version_name%-*}"* || true
+  rm -rf "versioned_sidebars/version-${new_version_name%-*}"*-sidebars.json || true
```
Comment (tobim) [inline scripts/prepare-release:82]: FYI: This modification is a follow up to #3278. It is a workaround for idiosyncratic behavior of zsh, which fails on a globbed `rm` if no matches are found.

Comment (mavam) [reply scripts/prepare-release:82]: This should be a comment in the code.

**Comment by mavam** [comment]
@tobim could you explain how a reviewer should look at the 9k LOC diff?

**Comment by tobim** [comment]
Almost all of this is created by copying from next to the versioned directories. None of the content is actually new.

**Review by mavam — APPROVED** [review]

**Comment by mavam** [comment]
So basically you just invoked the usual script that rotated the docs?

File: scripts/prepare-release (hunk 1 above)
Comment (tobim) [reply scripts/prepare-release:82]: I'll follow that up with a separate PR. The script should add a PR description as well.

**Comment by tobim** [comment]
> So basically you just invoked the usual script that rotated the docs?

Precisely.

**Pull request merged**